config/routes.rb

```
@@ -493,6 +495,60 @@
   lookup_user
 ].freeze
 
+def redirect_old_crud_actions(
+  old_controller: "",
+  new_controller: old_controller&.pluralize,
+  model: new_controller&.singularize,
+  actions: [:create, :destroy, :edit, :index, :show]
+)
+  return if old_controller.blank?
```

Might be better to throw an error than silently return.